.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
```

Comment:
I needed to split this out because GitHub actions do not report the branch cleanly like Travis did in the `TRAVIS_BRANCH` environment variable. The closest actions has is `GITHUB_REF` but that changes meaning and format between [pushes](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#push) & [releases](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#release).

Rather than try and infer it from the format of GITHUB_REF, I modified the depy-docs script to take an argument for the version. Then, I created this flow to deploy the docs on commits to master.

Reply:
I like this way. Maybe add a comment in top of this wokflow, something like

\# This workflow publishes https://chartjs.org/doscs/master after every commit